File: 1-Iniciante/Cedulas/1018.c
#include <stdio.h>

int main() {

    int n, analog;
    scanf ("%d", &n);
    analog = n;
    int qnt[7] = {0, 0, 0, 0, 0, 0, 0};
    if (n >= 100) {
        qnt[0] = n / 100;
        n -= 100 * (n / 100);
    };
    if (n >= 50) {
        qnt[1] = n / 50;
        n -= 50 * (n / 50);
    };
    if (n >= 20) {
        qnt[2] = n / 20;
        n -= 20 * (n / 20);
    };
    if (n >= 10) {
        qnt[3] = n / 10;
        n -= 10 * (n / 10);
    };
    if (n >= 5) {
        qnt[4] = n / 5;
        n -= 5 * (n / 5);
    };
    if (n >= 2) {
        qnt[5] = n / 2;
        n -= 2 * (n / 2);
    };
    if (n >= 1) {
        qnt[6] = n;
    };
    printf ("%d\n", analog);
    printf ("%d nota(s) de R$ 100,00\n", qnt[0]);
    printf ("%d nota(s) de R$ 50,00\n", qnt[1]);
    printf ("%d nota(s) de R$ 20,00\n", qnt[2]);
    printf ("%d nota(s) de R$ 10,00\n", qnt[3]);
    printf ("%d nota(s) de R$ 5,00\n", qnt[4]);
    printf ("%d nota(s) de R$ 2,00\n", qnt[5]);
    printf ("%d nota(s) de R$ 1,00\n", qnt[6]);

    return 0;
}

File: 1-Iniciante/1018-Cedulas/1018.py
n = int(input())
analog = n
ceds = {'100': 0, '50': 0, '20': 0, '10': 0, '5': 0, '2': 0, '1': 0}
if n >= 100:
    ceds['100'] = n // 100
    n -= 100 * (n // 100)
if n >= 50:
    ceds['50'] = n // 50
    n -= 50 * (n // 50)
if n >= 20:
    ceds['20'] = n // 20
    n -= 20 * (n // 20)
if n >= 10:
    ceds['10'] = n // 20
    n -= 10 * (n // 10)
if n >= 5:
    ceds['5'] = n // 5
    n -= 5 * (n // 5)
if n >= 2:
    ceds['2'] = n // 2
    n -= 2 * (n // 2)
if n >= 1:
    ceds['1'] = n
print (analog)
print (ceds['100'], "nota(s) de R$ 100,00")
print (ceds['50'], "nota(s) de R$ 50,00")
print (ceds['20'], "nota(s) de R$ 20,00")
print (ceds['10'], "nota(s) de R$ 10,00")
print (ceds['5'], "nota(s) de R$ 5,00")
print (ceds['2'], "nota(s) de R$ 2,00")
print (ceds['1'], "nota(s) de R$ 1,00")